WrightTools/kit/_ini.py

```diff
@@ -107,7 +108,7 @@ def has_section(self, section):
         self.config.read(self.filepath)
         return self.config.has_section(section)
 
-    def read(self, section, option):
+    def read(self, section, option) -> str:
```

Comment:
Does not always return string, really quite a large union, honestly

Reply:
ok, I'll just remove the hint